assets/openapi/api-b2b-v3.yaml

```diff
@@ -1340,14 +1344,27 @@ components:
                   format: int
                   minimum: 0
                   maximum: 10
-       
         _embedded:
           type: object
+      required : [_links]
   
     MonitorId:
       type: string
       description: Uniq monitor ID returned when the monitor was created succefully.
   
+    MaintenanceBase:
+      type : object
+      description: Expresses next Maintenance details. Such as number of days and the mileage until the next Maintenance. The value is negative if the next maintenance (day or mileage) is already passed. At least one of the following maintenance information will be provided.
+      title: Maintenance
+      properties:
+        vin:  
+          $ref: '#/components/schemas/Vin'
+        daysBeforeMaintenance:
+          type: integer
+        mileageBeforeMaintenance:
+          type: integer
+      required: [createdAt, vin] 
+    
     MaintenanceObj:
```

Comment:
MaintenanceObj should simply extends  MaintenanceBase, updateAt and createdAt . So why redefine other properties instead of extend MaintenanceBase?